pkg/cephfs/controllerserver.go

```diff
@@ -44,6 +44,34 @@ var (
 	mtxControllerVolumeID = keymutex.NewHashed(0)
 )
 
+// ExpandVolume expand CephFS Volumes on demand based on resizer request
+func (cs *ControllerServer) ControllerExpandVolume(ctx context.Context, req *csi.ControllerExpandVolumeRequest) (*csi.ControllerExpandVolumeResponse, error) {
+
+	if err := cs.validateExpandVolumeRequest(req); err != nil {
+		klog.Errorf("ControllerExpandVolumeRequest validation failed: %v", err)
+		return nil, err
+	}
+
+	volID := req.GetVolumeId()
+	secret := req.GetSecrets()
```

Comment:
The resizer secret [name and namespace](https://github.com/kubernetes-csi/external-resizer/blob/c398721342811a9b47f8b96ff7112288e57d465b/pkg/resizer/csi_resizer.go#L40) are specialized, which in turn would need the same in the storageclass. Request update of documentation and examples to reflect the same.